test/cron_parser_test.dart

```diff
@@ -6,9 +6,9 @@ import '../lib/cron_parser.dart';
 
 void main() {
   DateTime normalizedDate(
-      [DateTime dateTime, String locationName = "Europe/Berlin"]) {
+      [DateTime? dateTime, String locationName = "Europe/Berlin"]) {
     var location = getLocation(locationName);
-    TZDateTime date = dateTime == null ? TZDateTime.now(location) : dateTime;
+    TZDateTime date = dateTime == null ? TZDateTime.now(location) : dateTime as TZDateTime;
     return TZDateTime.from(
```

Comment:
This cast is not good. Please change the return type.

Reply:
I think this is a change in Dart, because without `as TZDateTime` you will get this issue:
```
A value of type 'DateTime' can't be assigned to a variable of type 'TZDateTime'.
Try changing the type of the variable, or casting the right-hand type to 'TZDateTime'.
```

(This change was also added by the dart migration tool)

Reply:
yes because normalizedDate returns type DateTime. I would recommend to change this function to use and return TZDateTime instead of  DateTime

Reply:
Agree, I changed the return type of `normalizedDate` to `TZDateTime` in ee353df6010b8ae577eb1a0378a3421a6fa2f326